x-pack/plugins/fleet/server/services/epm/archive/parse.ts

```diff
@@ -324,7 +324,7 @@ export function parseAndVerifyDataStreams(opts: {
       streams: manifestStreams,
       elasticsearch,
       ...restOfProps
-    } = manifest;
+    } = expandDottedObject(manifest);
```

Comment:
Assuming this is where the fix from the resolved conversation below wound up being? One thing I've always found confusing is our support for both dotted and nested fields. We've had bugs come up around this before, as I recall. Is this just a matter of historical context that we support both schemes in package manifests?

Reply:
Yes this was the fix.

I have found this to be a common problem with YAML, not just in package-spec, but also in using helm charts. I think we should try to support this notation as there isn't a strong motivation not to from my perspective.